src/iperf_api.c

```diff
@@ -2305,6 +2305,7 @@ iperf_reset_test(struct iperf_test *test)
     test->settings->burst = 0;
     test->settings->mss = 0;
     test->settings->tos = 0;
+    test->settings->authtoken = NULL;
```

Comment:
Isn't this potentially a leak?  I'd do:

```
free(test->settings->authtoken);
test->settings->authtoken = NULL;
```

remembering that `free(NULL);` is a safe (but inefficient) operation.

Reply:
Yep, you're right...I should have caught that.  (The hazards of working on both C and Java code at the same time.)  Thanks for the correction.  I usually do something like:

```
if (test->settings->authtoken) {
    free(test->settings->authtoken);
    test->settings->authtoken = NULL;
}
```
I'm guessing there's no easy way short of unmerging to tell GitHub to re-open a pull request, right?

Reply:
Don't know if there is or not.  What happens if you click on Revert?

Reply:
```Sorry, this pull request couldn’t be reverted automatically. It may have already been reverted, or the content may have changed since it was merged.```

I know why...a line of code I changed just above this is preventing the reverse diff from applying.  I'll just fix this with a new PR...I'd like you to look that over if you don't mind.  Thanks!

Reply:
sorry for the issue! So, could I help with the new PR?

Reply:
@ralcini: No worries.  The commit you just mentioned is (I think) the same as what I did for PR #652.  Want to give that a review?  Thanks!

Reply:
@bmah888 yes, it is!